core/templates/dev/head/domain/suggestion/SuggestionObjectFactory.ts

```diff
@@ -17,12 +17,25 @@
    domain objects.
  */
 
-var oppia = require('AppInit.ts').module;
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-oppia.factory('SuggestionObjectFactory', [function() {
-  var Suggestion = function(
-      suggestionType, suggestionId, targetType, targetId, status, authorName,
-      stateName, newValue, oldValue, lastUpdated) {
+export class Suggestion {
+  suggestionType: string;
+  suggestionId: string;
+  targetType: string;
+  targetId: string;
+  status: string;
+  authorName: string;
+  stateName: string;
+  newValue: string;
+  oldValue: string;
+  lastUpdated: number;
```

Comment:
This should be Date?

Reply:
The corresponding spec file has it as a number.